### PR TITLE
Creating a default set of floating point exceptions

### PR DESCRIPTION
**Pull request by af1990**
Adding a simple function defining the default set of FPE we want to check.

### CODE REVIEW

File: unitTests/CMakeLists.txt [hunk 1]
```diff
@@ -87,6 +87,11 @@ blt_add_executable( NAME testFloatingPointExceptions
                     OUTPUT_DIR ${TEST_OUTPUT_DIRECTORY}
                     DEPENDS_ON gtest lvarray ${lvarray_dependencies} )
 
+# Need to avoid optimization to catch invalid operations
+if( APPLE AND ${CMAKE_CXX_COMPILER} MATCHES "clang" )
```
Comment (corbett5) [inline unitTests/CMakeLists.txt:91]: Ahh is this because APPLE + "clang" doesn't respect the no-inline attribute?

**Review by corbett5 — APPROVED** [review]
Thanks!

File: unitTests/CMakeLists.txt (hunk 1 above)
Comment (af1990) [reply unitTests/CMakeLists.txt:91]: I think so ... I am not a Mac expect!

**Review by rrsettgast — APPROVED** [review]

**Review by castelletto1 — APPROVED** [review]
Looks good to me. Thanks for the fixes!

**Pull request merged**